src/app.js

```diff
@@ -151,7 +151,7 @@ export function app(props, container) {
       var oldValue = i === "value" || i === "checked" ? element[i] : oldProps[i]
 
       if (value !== oldValue) {
-        value !== oldValue && setElementProp(element, i, value, oldValue)
```

Comment:
Leaving this line but removing the whole `if` would have had the same effect (minus 2 bytes perhaps? ⛳️) but it doesn't really conform with the style of other binary checks in the codebase. I am happy with either really, up to you @JorgeBucaran

Reply:
@lukejacksonn Actually uglifyjs already takes care of it. This is the minified result without the repeated code.

```js
for (var r in l(n, t)) {
  var o = t[r],
    i = "value" === r || "checked" === r ? e[r] : n[r]
  o !== i && a(e, r, o, i)
}
```

Reply:
Well what do you know! Good job uglify 🥇

Reply:
So, can we replace all binary check tricks with a real `if` block or in some cases this is optimised by hand ?

Reply:
@Swizz Yup, that's why I keep all the `if`s in there. We don't need to optimize those. 🎉